lib/sitemap-item.d.ts

```diff
@@ -1,77 +1,115 @@
 import builder = require('xmlbuilder');
+
+export declare interface NewsItem {
+  publication: {
+    name: string,
+    language: string
+  },
+  genres: string,
+  publication_date: string,
+  title: string,
+  keywords: string,
+  stock_tickers: string
+}
+
+export declare interface SitemapImg {
+  url: string,
+  caption: string,
+  title: string,
+  geoLocation: string,
+  license: string
+}
+
+export declare type yesno = 'yes' | 'no'
+export declare type allowdeny = 'allow' | 'deny'
+export declare type ChangeFrequency = 'always'|'hourly'|'daily'|'weekly'|'monthly'|'yearly'|'never'
```

Comment:
Just spacing between pipes `|`. In previous 2 types (`yesno` and `allowdeny `) you have some.